devicon.json

```
@@ -8092,6 +8092,24 @@
         "color": "#68217A",
         "aliases": []
     },
+    {
+        "name": "vitejs",
+        "tags": [
+            "react", 
```

vite.js isn't exclusive to react, so I think we should replace it with javascript, and also add the build tag
```suggestion
            "javascript",
            "build"
```